azurerm/internal/services/storage/tests/resource_arm_storage_share_file_test.go

```diff
@@ -0,0 +1,316 @@
+package tests
+
+import (
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"testing"
+
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/services/storage/parse"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/terraform"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/acceptance"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/clients"
+)
+
+func TestAccAzureRMStorageShareFile_basic(t *testing.T) {
```

Comment:
As we are slowly updating all the tests might be a good idea to switch this over now
```suggestion
func TestAccStorageShareFile_basic(t *testing.T) {
```

Reply:
As well as use the test shim